.github/workflows/build-image.yml

```diff
@@ -61,7 +63,7 @@ jobs:
                 file: ./src/Dfe.PlanTech.Web/Dockerfile
                 build-args: COMMIT_SHA=${{ needs.set-env.outputs.checked-out-sha }}
                 tags: |
-                    ghcr.io/dfe-digital/${{ env.DOCKER_IMAGE }}:${{ needs.set-env.outputs.branch }}
-                    ghcr.io/dfe-digital/${{ env.DOCKER_IMAGE }}:sha-${{ needs.set-env.outputs.checked-out-sha }}
-                    ghcr.io/dfe-digital/${{ env.DOCKER_IMAGE }}:latest
+                    ${{ env.GITHUB_CONTAINER_REGISTRY }}/${{ env.ORG_NAME }}/${{ env.DOCKER_IMAGE }}:${{ needs.set-env.outputs.branch }}
```

Comment:
Interesting, I think we can leave it for now because it can only be triggered on workflow_call and only matrix deploy will run it, but if we ever trigger it on different branches, or add say workflow_dispatch etc, then we could add some filtering to check if it's a valid string et al at that time. Cheers!